refinery/file_store/utils.py

```diff
@@ -2,11 +2,28 @@
 
 from django.conf import settings
 from django.core.files.storage import FileSystemStorage
+from django.utils.crypto import get_random_string
 from django.utils.deconstruct import deconstructible
+from django.utils.text import get_valid_filename
 
 from storages.backends.s3boto3 import S3Boto3Storage
 
 
+class S3MediaStorage(S3Boto3Storage):
```

Comment:
Would it make sense to have an abstract base for these two with a placeholder get_available_name that raises NotImplementedError or something of the like?

I also don't envision there being any other storage backends for now so that might be overkill

Reply:
Their common parent is the [Storage class provided by Django](https://docs.djangoproject.com/en/1.8/_modules/django/core/files/storage/#Storage)